rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -161,3 +161,80 @@ disabling stored fields removes fetch sub phases:
   - match: { hits.hits.0._index: test }
   - match: { profile.shards.0.fetch.debug.stored_fields: [] }
   - is_false: profile.shards.0.fetch.children
+
+---
+dfs knn vector profiling:
```

Comment:
Can you profile with the dfs phase enabled without knn? We shouldn't see anything, but I want to be extra paranoia that it runs and doesn't crash.

Reply:
I see you did it below but I didn't realize because of the name.

Reply:
I'll split these up.